### ISSUE TITLE
Some commonly-reported diagnostic results at suggestion level cannot be disabled

### ISSUE THREAD

**Issue by SunnieShine** [dotnet/roslyn#69829]
**Version Used**: 

VS 17.8p1

**Steps to Reproduce**:

```csharp
[SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "<Pending>")]
private static void UnusedMethod(int unusedParameter)
{
}
```

**Diagnostic Id**:

Some diagnostic results at suggestion level (Maybe result IDs starting with `IDE` like `IDE0060`? I don't test for all diagnostic results, but some commonly-reported can be reproduced, e.g. IDE0051, IDE0060, IDE0011 and so on).

**Expected Behavior**:

If I use `SuppressMessageAttribute` and `#pragma warning disable`, the suggestions can be disabled and no longer be displayed.

**Actual Behavior**:

It is still displayed no matter how I disable them:

![image](https://github.com/dotnet/roslyn/assets/23616315/bab66fac-870b-4b59-9cd5-02036d21b12e)

![image](https://github.com/dotnet/roslyn/assets/23616315/bdead729-2d46-4a27-b466-d204ab6cbbdf)

![image](https://github.com/dotnet/roslyn/assets/23616315/02b6e76c-14c5-4b42-97e5-1cd67b57cf75)

**Comment by huoyaoyuan** [comment]
In what scope do you want to disable it? how about using [globalconfig](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-files#global-analyzerconfig)?

**Comment by SunnieShine** [comment]
@huoyaoyuan I just want to disable it in this file...

**Comment by genlu** [comment]
@mavasani Any suggestions here?

**Comment by mavasani** [comment]
This is a duplicate of https://developercommunity.visualstudio.com/t/SuppressMessage-attribute-is-being-ign/10439130.
It was fixed in VS2022 17.8 Preview2 with #69492
It was backported to VS2022 17.7 release with #69517

**Comment by SunnieShine** [comment]
Thank you!
